epochStart/bootstrap/process.go

```diff
@@ -878,6 +881,11 @@ func (e *epochStartBootstrap) createRequestHandler() error {
 	return err
 }
 
+func (e *epochStartBootstrap) setEpochStartMetrics() {
+	e.statusHandler.SetUInt64Value(core.MetricNonceAtEpochStart, e.epochStartMeta.Nonce)
```

Comment:
shouldn't we also rewrite the metrics for `erd_nonces_passed_in_current_epoch` and `erd_rounds_passed_in_current_epoch` ?

Reply:
No, those are calculated when calling the API endpoint. So it is ok

Reply:
ok